.buildkite/scripts/generate_metricbeat_pipeline.sh

```diff
@@ -33,7 +29,7 @@ if are_conditions_met_mandatory_tests; then
         agents:
           provider: "gcp"
           image: "${IMAGE_UBUNTU_X86_64}"
-          machineType: "c2-standard-16"
+          machineType: "c2d-highcpu-16"
```

Comment:
Since this value is hardcoded in a few places, maybe a variable could help.

Reply:
I could be changed again https://elastic.slack.com/archives/C0522G6FBNE/p1707140315980879

Reply:
done